### PR TITLE
Merge Dev into master

### PR DESCRIPTION
**Pull request by aymanhab**
Development branch has many changes on it, this merges these changes to master

### CODE REVIEW

**Comment by aymanhab** [comment]
Per our discussion yesterday, this brings master in sync. with Dev so that new changes can be isolated/tested with better granularity.

File: src/backend/readme.md
Comment (AlbertoCasasOrtiz) [inline src/backend/readme.md:1]: I think this file looks exactly the same as the one called README in uppercase, is there any reason we have both?

File: src/components/Components/DropFile.tsx [hunk 1]
```diff
@@ -112,20 +116,25 @@ const FileDropArea = observer(() => {
                 const api_url = 'https://eudfxg3a9l.execute-api.us-west-2.amazonaws.com/dev/'
                 axios.post(api_url, data).then(response => {
                   const gltf_url = response.data['url']; .replace(/\.\w+$/, '.gltf')
-                  appState.setCurrentModelPath(gltf_url); */
+                  appState.setCurrentModelPath(gltf_url); 
+                  CognitoIdentityServiceProvider.6jlm2jeibh9aqb0dg34q2uf8pu.ayman1234.userData*/
```
Comment (AlbertoCasasOrtiz) [inline src/components/Components/DropFile.tsx:120]: This looks like some kind of ID or sensible information. Should we remove it?

File: src/state/ViewerState.tsx [hunk 2]
```diff
@@ -78,6 +85,19 @@ class ViewerState {
     }
     setIsLoggedIn(newState: boolean) {
         this.isLoggedIn = newState
+        if (this.isLoggedIn){
+            // Cache user_uuid until logout
+            const userName = localStorage.getItem('CognitoIdentityServiceProvider.6jlm2jeibh9aqb0dg34q2uf8pu.LastAuthUser');
+            const storedDataString = localStorage.getItem('CognitoIdentityServiceProvider.6jlm2jeibh9aqb0dg34q2uf8pu.'+userName+'.userData');
```
Comment (AlbertoCasasOrtiz) [inline src/state/ViewerState.tsx:91]: Same here with the CognitoServiceProvider, are those ID ok here? I think it is but just want to make sure it is not sensible information.

**Review by AlbertoCasasOrtiz — CHANGES_REQUESTED** [review]
Everything looks good to me. Just a few comments on the code.

I have tested dev with a few files and everything seems to work fine.

File: src/backend/readme.md
Comment (aymanhab) [reply src/backend/readme.md:1]: Sure, we should remove one and keep the uppercase version per convention

File: src/components/Components/DropFile.tsx (hunk 1 above)
Comment (aymanhab) [reply src/components/Components/DropFile.tsx:120]: Please remove, good catch, same for the other comment.

**Comment by aymanhab** [comment]
I believe I responded to your feedback @AlbertoCasasOrtiz  please let me know if we should merge. Thank you

**Review by AlbertoCasasOrtiz — APPROVED** [review]
Everything Looks good, I just changed the name of the README_1.md file to README.md.

**Comment by aymanhab** [comment]
Thanks much @AlbertoCasasOrtiz 👍 Merging

**Pull request merged**